pkg/microservice/aslan/core/environment/service/environment.go

```diff
@@ -512,7 +507,7 @@ func UpdateProduct(existedProd, updateProd *commonmodels.Product, renderSet *com
 
```

Comment:
should we update the above logic of deleting services directly when releated service templates are deleted?